package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "wordpress-importer",
+  "version": "1.0.0",
+  "description": "https://github.com/Automattic/wordpress-importer",
+  "main": "index.js",
+  "dependencies": {
+    "@wordpress/components": "^7.0.8",
+    "@wordpress/compose": "^3.0.1",
+    "react": "16.6.3",
+    "react-dom": "16.6.3",
+    "source-map-loader": "^0.2.4"
+  },
+  "scripts": {
+    "build": "wp-scripts build",
```

Comment:
`@wordpress/scripts` was updated and published to npm to support `build` command:

https://unpkg.com/@wordpress/scripts@3.0.0/scripts/build.js

Reply:
Awesome! Thanks heaps for updating us here @gziolo!
We've been exploring moving the project to Jetpack so might end up abandoning this approach but I'm looking forward to using `wp-scripts` in the future! <3

Reply:
That would make a lot of sense to re-use Jetpack configs as well. Whatever works for you :)